React/tic-tac/src/app/components/withIsLoading/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import style from './style.scss';
+
+function LoaderHOC(WrappedComponent) {
+  return function wrapper(props) {
```

Comment:
```suggestion
  return function wrapper({ userLoading }) {
```

Reply:
Create the constant because in the component it calls '... props' ---> <WrappedComponent {...props} />